### PR TITLE
Headphones master button working!

### PR DESCRIPTION
**Pull request by fkbreitl**
I got it working!
Cheers!

### CODE REVIEW

File: Pioneer-DDJ-200-scripts.js [hunk 1]
```diff
@@ -38,4 +41,15 @@ DDJ200.seek = function (channel, control, value, status, group) {
...
+
+DDJ200.headmix = function (channel, control, value, status, group) {
+    if (value == 0) { return; }
+    DDJ200.headMix_val = 1 - DDJ200.headMix_val;
+    engine.setValue("[Master]", "headMix", 2 * DDJ200.headMix_val - 1);
+    // headMix knob has values from -1 to 1
+
+    DDJ200.headMix_LED = 0x7F - DDJ200.headMix_LED;
+    midi.sendShortMsg(0x90+channel, control, DDJ200.headMix_LED);  // headMix
+    //midi.sendShortMsg(0x96, 0x63, 0x7F);  // headMix button
```
Comment (dan-giddins) [inline Pioneer-DDJ-200-scripts.js]: If this comment is no longer needed, could you please remove it?

Comment (dan-giddins) [reply Pioneer-DDJ-200-scripts.js]: Apart from that the PR looks good!

Comment (fkbreitl) [reply Pioneer-DDJ-200-scripts.js]: > If this comment is no longer needed, could you please remove it?

Which comment do you mean? I thought both comments are useful.

Comment (dan-giddins) [reply Pioneer-DDJ-200-scripts.js]: @fkbreitl just a bump on this

**Review by fkbreitl — COMMENTED** [review]
All good

File: Pioneer-DDJ-200-scripts.js (hunk 1 above)
Comment (dan-giddins) [reply Pioneer-DDJ-200-scripts.js]: @fkbreitl It looks much better now, thanks. I thought it was an old left over line of code that you had commented out, but now I see it is useful in explaining what values are going to be passed into sendShortMsg()

**Review by dan-giddins — APPROVED** [review]

**Pull request merged**